x/vm/types/params.go

```diff
@@ -17,15 +17,18 @@ import (
 )
 
 var (
+	// DefaultEVMDenom is the default value for the evm denom
+	DefaultEVMDenom = "atest"
+	// DefaultEVMChainID is the default value for the evm chain ID
+	DefaultEVMChainID = "cosmos_262144-1"
+	// DefaultEVMDecimals is the default value for the evm denom decimal precision
+	DefaultEVMDecimals uint64 = 18
```

Comment:
I think this stuff is probably best specified via the chain itself--like evmd could use these variables, but other chains should just get these as defaults.